src/allure-nunit-tests/allureConfig.json

```diff
@@ -1,13 +1,41 @@
 ﻿{
-	"allure": {
-		"directory": "allure-results",
-		"links": [
-			"https://github.com/nunit/docs/issues?utf8=✓&q={issue}",
-			"https://example.org/{tms}",
-			"{link}"
-		],
-		"brokenTestData": [
-			"System.Exception"
-		]
-	}
+  "allure": {
+    "directory": "allure-results",
+    "links": [
+      "{link}",
+      "{issue}",
+      "{tms}"
+    ]
+  },
+  "specflow": {
```

Comment:
There's no Specflow in this project, so why did you add this?

Reply:
Quick question,
I noticed you extended AllureConfiguration to include some customisations. I need to do the same thing myself.

I think these customizations should live in an nunit configuration section like SpecFlow as they only affect the generation of NUnit tests.

Backward compatability is an issue and a fallback to the old location is an option.

What do you think?

Reply:
I think it can go to docs, not. Please keep original allureConfig.json except `brokenTestData`